packages/components/src/internal/components/UserAvatars.tsx

```diff
@@ -1,30 +0,0 @@
-import { User } from '@labkey/api';
-import React, { FC, memo } from 'react';
-
-// This is due to original implementation coming from ELN which serializes users in a limited fashion, we can convert
-// this to just be User when we update ELN to serialize Users.
-type AvatarUser = Pick<User, 'avatar' | 'displayName' | 'id'>;
-
-interface UserAvatarProps {
-    id: number;
-    avatar: string;
-    displayName: string;
-}
-
-export const UserAvatar: FC<UserAvatarProps> = memo(({ avatar, displayName, id }) => (
-    <img key={id} className="user-avatar" src={avatar} alt={displayName} title={displayName} />
```

Comment:
Should also remove the `.scss` declarations/usages of `"user-avatar"`.

Reply:
Ah, yes, good catch